msu/systems/registry/mod_sources/mod_source_githubtags.nut

```diff
@@ -0,0 +1,28 @@
+::MSU.Class.ModSourceGitHubTags <- class extends ::MSU.Class.ModSource
+{
+	static ModSourceDomain = ::MSU.Class.RegistrySystem.ModSourceDomain.GitHubTags;
+	static Regex = regexp("https:\\/\\/github\\.com\\/([-\\w]+)\\/([-\\w]+)/?.*");
+	static BadURLMessage = "A link must point into a Github repository repository, e.g. 'https://github.com/Suor/battle-brothers-mods/something/else/'";
+	static Icon = "github";
+	Prefix = ""
+
```

Comment:
The constructor based on the previous _opts comments would be something like this:
```
	constructor( _url, _opts = null )
	{
		base.constructor(_url, _opts);
		if (_opts != null && "Prefix" in _opts)
		{
			this.Prefix = _opts.Prefix;
		}
	}
```

Reply:
Then one will pass `{prefix = "autopilot-new-"}` and will happily debug it for an hour instead of getting an error instantly now. Also will need to repeat for each key. 

Why fight against the language instead of using it?

Reply:
You can still have erroring if you want. Yes, you will repeat for each key, which amounts to one key at this time, and makes it clear that this key is set in this space for this one mod source.

Reply:
Okay look, it's fine for me, just add some clarifying comment on the Prefix member that it's set via the _opts parameter in the base class.

Reply:
Added a comment.